runtime/builtin_types.go

```diff
@@ -190,6 +190,29 @@ func builtinAbs(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
 	return Abs(f, args[0])
 }
 
+func builtinAll(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+	if raised := checkFunctionArgs(f, "all", args, ObjectType); raised != nil {
+		return nil, raised
+	}
+	iter, raised := Iter(f, args[0])
+	if raised != nil {
+		return nil, raised
+	}
+	o, raised := Next(f, iter)
+	for ; raised == nil; o, raised = Next(f, iter) {
+		if ret, raised := IsTrue(f, o); raised != nil {
+			return nil, raised
+		} else if !ret {
```

Comment:
... declaring outside the if is the style I've *tried* to follow in the source code... :D